src/libsyntax/ast.rs

```diff
@@ -178,9 +178,11 @@ pub struct LifetimeDef {
     pub bounds: Vec<Lifetime>
 }
 
-/// A "Path" is essentially Rust's notion of a name; for instance:
-/// std::cmp::PartialEq  .  It's represented as a sequence of identifiers,
+/// A "Path" is essentially Rust's notion of a name.
```

Comment:
Can you add an empty line here (well, one that starts with `///` actually), so rustdoc can pick this first sentence for use as a short description?